app/support/date_formatter.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DateFormatter
+  include ActionView::Helpers::DateHelper
+
+  def distance_of_time_in_words_from_now(datetime)
+    distance_of_time_in_words(Time.zone.now, datetime)
```

Comment:
`distance_of_time_in_words` is inherited from [`ActionView::Helpers::DateHelper`](http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words)